### PR TITLE
#2615 Corrections Dash P3: Resume Correction Drafts

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#2615

*Description of changes:*
* Implements resuming draft correction filings.
* Implements tags on the original filings corrected and when there is a pending correction.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
         <v-expansion-panel-header class="filing-item-toggle">
           <div class="list-item">
             <div class="filing-label">
-              <h3>{{item.title}}</h3>
+              <h3>{{item.title}} {{applyCorrectionTag(item)}}</h3>
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue]: Nice use of a getter!

File: src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -133,91 +154,115 @@
...
           </div>
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-
-          <div v-if="isCorrection(task)" data-test-class="correction-comment" class="todo-list-detail">
+          <div v-if="isCorrection(task)">
+            <div v-if="isDraft(task)" data-test-class="correction-draft" class="todo-list-detail">
+              <p class="list-item__subtitle">
+                This filing is incomplete. The payment appears to have been unsuccessful for some reason.
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue]: This sentence may or may not be true...
- if the staff saved a draft to resume later, this statement is false
- if our API returned with a filing error and saved a draft, this statement is true

You probably need to discuss this with Scott, unless you feel pre-emptive and want to make the second sentence conditional for example.

Comment (cameron-freshworks) [reply src/components/Dashboard/TodoList.vue]: K, will discuss with Scott and confirm design, thanks!

Comment (severinbeauvais) [reply src/components/Dashboard/TodoList.vue]: I see the Draft text is updated in a future commit... so this conversation is resolved!

**Pull request merged**